tests/test_pkce.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
This file should now be `test__pkce.py`, which looks funny to me but fits a pattern that we have not yet determined to be a bad practice to be reformed. :-P